res/flow-capture.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: netobserv-cli
+  namespace: netobserv-cli
+  labels:
+    app: netobserv-cli
+spec:
+  selector:
+    matchLabels:
+      app: netobserv-cli
+  template:
+    metadata:
+      labels:
+        app: netobserv-cli
+    spec:
+      serviceAccountName: netobserv-cli
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+      - name: netobserv-cli
+        # image: quay.io/netobserv/netobserv-ebpf-agent:main
+        image: quay.io/jpinsonn/netobserv-ebpf-agent:flow-stream-1
```

Comment:
https://github.com/netobserv/netobserv-ebpf-agent/pull/224 needs to be merged first

Reply:
I wonder if u can reuse the current gRPC exporter from ebpf agent instead of creating new TCP exported ?

Reply:
Would that work with `oc port-forward` ?

Reply:
gRPC uses TCP so I would think so